spec/commands/newgem_spec.rb

```diff
@@ -499,20 +499,20 @@ def create_temporary_dir(dir)
       end
 
       it "builds spec skeleton" do
-        expect(bundled_app("test-gem/test/test_test/gem.rb")).to exist
```

Comment:
this test illustrate this PR!
`bundle gem test-gem` would generete:
before: `test-gem/test/test_test/gem.rb`
after: `test-gem/test/test/gem_test.rb`